### ISSUE TITLE
Add translate support...

### ISSUE THREAD

**Issue by ziceptor** [Xwaffle1/UniversalMarket#8]
Chat message. GUI. Support currency plug-in economy. Other.

**Comment by ziceptor** [comment]
Translation support will make the plugin attractive to many users.

**Comment by demon57730** [comment]
I agree, because I am a French server but I and other players do not speak much English so that the messages and words in the market are not all understood.

so it would be nice to create a message.conf file (as other plugins do) to be able to change the text
